### PR TITLE
fix: labels follows terraform convention

### PR DESCRIPTION
**Pull request by AlexanderBrevig**
https://github.com/entur/terraform-google-init/blob/8b8051e07af0174d15108f763004bf58516c2868/modules/init/outputs.tf#L6-L15

```terraform
output "labels" {
  description = "Labels for use on managed resources (i.e. Kubernetes resources)."
  value = {
    app         = local.app.name
    app_id      = var.app_id
    environment = var.environment
    owner       = local.app.owner
    team        = local.app.owner
  }
}
```

### CODE REVIEW

File: charts/common/templates/_helpers.tpl [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 {{- define "labels" }}
 app: {{ empty .Values.releaseName | ternary .Release.Name .Values.releaseName }}
-shortname: {{ .Values.shortname }}
```
Comment (AlexanderBrevig) [inline charts/common/templates/_helpers.tpl:7]: maybe someone already uses `shortname` and we should add it back?

Comment (kagkarlsson) [reply charts/common/templates/_helpers.tpl:7]: Yeah, maybe use a deprecation-window. Leave it for a couple of releases..

File: charts/common/templates/_helpers.tpl [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
-team: {{ .Values.team }}
+app_id: {{ .Values.shortname }}
+team: team-{{ .Values.team }}
```
Comment (kagkarlsson) [inline charts/common/templates/_helpers.tpl:8]: They might already use this aswell ?

Comment (kagkarlsson) [reply charts/common/templates/_helpers.tpl:8]: If we want to change it, maybe make it a breaking version and give a heads-up in slack

Comment (AlexanderBrevig) [reply charts/common/templates/_helpers.tpl:8]: Maybe we should keep `team` as is, and `shortname` and rename the new with some prefix like `entur_team` and `entur_app_id`. 
I think the most relevant case for these is in the cost reports.

**Pull request closed without merging**